src/erlfmt_scan.erl

```diff
@@ -178,12 +179,12 @@ last_node_string(#state{original = [First | _] = Tokens}) ->
     Location = erl_scan:location(First),
     {String, token_anno([{text, String}, {location, Location}])}.
 
+is_white_space({white_space, _, [$\n | _]}) -> true;
+is_white_space(_) -> false.
```

Comment:
This checks for newline specifically, not just any whitespace - this name is not that accurate

Reply:
Yes good catch.  I will change this.